iac-common/src/main/java/org/sonar/iac/common/parser/grammar/LexicalConstant.java

```diff
@@ -17,7 +17,7 @@
  * along with this program; if not, write to the Free Software Foundation,
  * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
  */
-package org.sonar.iac.terraform.parser.grammar;
+package org.sonar.iac.common.parser.grammar;
```

Comment:
I fear there may be some Lexical differences between HCL/Terraform and DockerFile.
I found one about comments :
- HCL support #, // and /* */
- Docker support only #

So not sure if we can completely move this Lexical class in common.
Maybe we can do so for mutual elements, and have two child class (HclLexicalConstant and DockerLexicalConstant) inheriting from it and completing with their particularity.

Reply:
Nils suggested moving this class there. But let's create `DockerLexicalConstant` for Docker specific things.

Reply:
Yeah, we can somehow create our own rule for comments in docker. It can be done directly in the DockerLexicalGrammar